style/scripts/taginfo_template.json

```diff
@@ -10,10 +10,59 @@
     "contact_email": "zelonewolf@gmail.com"
   },
   "tags": [
+    {
+      "key": "capital",
+      "value": "2",
+      "object_types": ["node"],
+      "description": "Marks the city with a national capital icon.",
+      "doc_url": "https://openmaptiles.org/schema/#capital",
+      "icon_url": "https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/style/icons/star_nation_capital.svg"
+    },
+    {
+      "key": "capital",
+      "value": "4",
+      "object_types": ["node"],
+      "description": "Marks the city with a state capital icon.",
+      "doc_url": "https://openmaptiles.org/schema/#capital",
+      "icon_url": "https://raw.githubusercontent.com/ZeLonewolf/openstreetmap-americana/main/style/icons/star_state_capital.svg"
+    },
+    {
+      "key": "claimed_by",
+      "object_types": ["way", "relation"],
+      "description": "Hides the boundary.",
+      "doc_url": "https://openmaptiles.org/schema/#claimed_by"
+    },
+    {
+      "key": "expressway",
+      "value": "yes",
+      "object_types": ["way"],
+      "description": "Highlights the road without affecting the zoom levels at which it appears.",
+      "doc_url": "https://openmaptiles.org/schema/#expressway"
+    },
+    {
+      "key": "maritime",
+      "value": "yes",
+      "object_types": ["way"],
+      "description": "Hides the boundary.",
+      "doc_url": "https://openmaptiles.org/schema/#maritime"
+    },
+    {
+      "key": "network",
+      "object_types": ["relation"],
+      "description": "Determines the design of the shield that marks roads carrying the route relation. An unrecognized value results in a generic shield.",
+      "doc_url": "https://openmaptiles.org/schema/#route_1"
+    },
     {
       "key": "ref",
       "object_types": ["relation"],
-      "description": "Populates shields that mark roads carrying the route relation."
+      "description": "Populates shields that mark roads carrying the route relation.",
+      "doc_url": "https://openmaptiles.org/schema/#route_1"
+    },
+    {
+      "key": "surface",
+      "object_types": ["way"],
+      "description": "Unpaved roads have an alternating dash pattern.",
+      "doc_url": "https://openmaptiles.org/schema/#surface"
```

Comment:
This documentation lists several specific values of `surface` that are considered “unpaved”. I refrained from hard-coding them here because the list could change out from under us.